### PR TITLE
[v1.4] Add ACI CNI 6033, 6041 variables

### PR DESCRIPTION
**Pull request by akhilesh-oc**
https://github.com/rancher/rancher/issues/44628

### CODE REVIEW

**Review by snasovich — DISMISSED** [review]

**Review by chiukapoor — APPROVED** [review]

**Comment by kinarashah** [comment]
@akhilesh-oc Could you re-run `go generate`? KDM PR is merged so we can update data.json with this PR before merging it.

**Comment by akhilesh-oc** [comment]
> @akhilesh-oc Could you re-run `go generate`? KDM PR is merged so we can update data.json with this PR before merging it.

Should we  wait till our changes get merged to dev-v2.8 for generating data ?
which URL should we update to generate data?

**Review by snasovich — APPROVED** [review]

**Review by chiukapoor — APPROVED** [review]

**Pull request merged**
